src/lib/components/Lock.svelte

```diff
@@ -42,7 +42,9 @@
 
 	const checkBalance = () => {
 		if (amountToLock) {
-			const bigNumValue = parseUnits(amountToLock.toString(), $selectedCyToken.decimals);
+			const bigNumValue = BigInt(
+				parseUnits(amountToLock.toString(), $selectedCyToken.decimals).toString()
```

Comment:
why do we need to do .toString() and then BigInt() it? the result of parseUnits is already a BigInt isn't it?